src/components/controllers/tutorial/style.module.css

```diff
@@ -18,12 +18,18 @@
 .tutorial {
 	position: absolute;
 	left: 0;
-	top: 96px;
+	top: var(--header-and-banner-height);
 	bottom: 0;
 	height: auto;
 	width: 100%;
 	overflow: hidden;
 
+	@media (max-height: 431px) {
```

Comment:
432px seemed like a reasonably safe value, just above the width of most phones (going by the device emulator in my devtools). The vast majority of phones should fall below this, giving them the little bit of extra height when rotated.